### DISCUSSION TITLE
VLC issue?

### DISCUSSION THREAD

**Discussion by chicken-nugget1104** [FunkinCrew/Funkin#2487] [Compiling help]
Error: Video.cpp
./src/hxcodec/openfl/Video.cpp(6): fatal error C1083: Cannot open include file: 'vlc/vlc.h': No such file or directory

**Comment by biomseed** [comment]
I'm pretty sure there's something else you have to install with hxcodec, i'm pretty sure it's vlc media player itself actually

**Reply by chicken-nugget1104** [reply]
> I'm pretty sure there's something else you have to install with hxcodec, i'm pretty sure it's vlc media player itself actually

i installed it did nothing

**Comment by lando07** [comment]
I found the issue: `vlc.h` comes from `libvlc-dev` and there's linker errors afterwards if you don't also install `libvlccore-dev` AND `libvlccore9`. When I installed them using `sudo apt install libvlc-dev libvlccore-dev libvlccore9` and then ran `lime test linux,` it built without the error.

**Reply by NotHyper-474** [reply]
In OP's case [they were on Windows](https://github.com/FunkinCrew/Funkin/issues/2484#issuecomment-2106335125)

**Reply by lando07** [reply]
ohhh well I fixed it on Linux, not sure how you can fix that on Windows.
